recipes/hatch-cython/recipe.yaml

```
@@ -0,0 +1,59 @@
+context:
+  name: hatch-cython
+  version: "0.5.1"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/joshua-auchincloss/${{ name|lower }}/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: de22dbbbaaded53ae6189f174f24f28f9649e04a05d9c8f629c751dcb59f2c61
+
+build:
+  script: ${{ PYTHON }} -m pip install --no-deps --no-build-isolation -vv .
+  number: 0
+  noarch: python
+
+requirements:
```

You will need a compiler:

```suggestion
requirements:
  build:
    - ${{ compiler('c') }}
    - ${{ compiler('cxx') }}
    - ${{ stdlib('c') }}
```